dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -1,26 +1,121 @@
-# Copyright (c) 2018 Peter Bigot Consulting, LLC
-# Copyright (c) 2019 Nordic Semiconductor ASA
+# Copyright (c) 2021, STMicroelectronics
```

Comment:
What ?

Reply:
Hi,
I have canceled the modification of this file.